app/scripts/main.js

```diff
@@ -4,6 +4,10 @@ import 'scripts/app.config.js';
 import 'scripts/app.run.js';
 import 'scripts/app_enforceAuth.js';
 
+// components
+import 'scripts/components/FormStatusModal/FormStatusModal';
+import 'scripts/components/FormStatusPopover/FormStatusPopover';
```

Comment:
I was going to suggest importing these where they are used. Although we're not really using the module system right and everything is a global... 

MPDX is making a module for each component https://github.com/CruGlobal/mpdx_web/blob/master/src/home/weeklyActivity/weeklyActivity.component.ts#L70 so at least it would throw an error if dependencies aren't included for each import.

Maybe we should leave it alone for now.

Reply:
I may address it in a separate pr in the future.